src/System.Net.Libuv/src/project.lock.json

```diff
@@ -3,10 +3,10 @@
   "version": 2,
   "targets": {
     "DNXCore,Version=v5.0": {
-      "System.Buffers/0.1.0-d103015-1": {
+      "System.Buffers/0.1.0-d7ileDaAe-1": {
```

Comment:
I'm not really sure...the PS script just gets the current date. Running it locally gives the correct values...